ios/Classes/PdftronFlutterPlugin.m

```diff
@@ -635,8 +635,10 @@ + (void)disableTools:(NSArray<id> *)toolsToDisable documentController:(PTDocumen
         if ([item isKindOfClass:[NSString class]]) {
             NSString *string = (NSString *)item;
             
-            if ([string isEqualToString:PTAnnotationEditToolKey]) {
-                // multi-select not implemented
+            if ([string isEqualToString:PTAnnotationEditToolKey] ||
+                [string isEqualToString:PTEditToolButtonKey] ||
+                [string isEqualToString:PTMultiSelectToolKey]) {
```

Comment:
I don't know what each of these buttons are....

Reply:
The first two, `PTAnnotationEditToolKey` and `PTEditToolButtonKey`, correspond to [`PTAnnotEditTool`](https://www.pdftron.com/api/ios/Classes/PTAnnotEditTool.html) internally, and they affect whether annotations can be edited or not. If they can't be edited, then it makes sense that multiple annot selection is turned off (this is same as React Native implementation).

The last one `PTMultiSelectToolKey` is for multiple annot selection and corresponds to [`PTAnnotSelectTool`](https://www.pdftron.com/api/ios/Classes/PTAnnotSelectTool.html) internally.